src/components/faqbox/index.js

```diff
@@ -1,26 +1,55 @@
-import React from 'react';
-import styled from 'styled-components'
-;
+import React from "react";
+import styled from "styled-components";
 const Box = styled.div`
-    background-color: white;
-    box-shadow: 2px 10px 30px rgba(0, 0, 0, 0.25);
-    border-radius: 7px;
-    text-align: left;
-    width: 20rem;
-    margin: 5rem;
-    padding: 2rem;
+  background-color: white;
+  box-shadow: 2px 10px 30px rgba(0, 0, 0, 0.25);
+  border-radius: 7px;
+  text-align: left;
+  width: 20rem;
+  margin: 5rem;
+  padding: 2rem;
+  font: Roboto;
+  font-style: normal;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-end;
+`;
+
+const Title = styled.div`
```

Comment:
In order to make our sites more readable/accessible, it is helpful to use semantic HTML tags. Here, h2 might work better.